csharp/extractor/Semmle.Extraction.CSharp.Standalone/Extractor.cs

```diff
@@ -33,7 +33,9 @@ private static IEnumerable<Action> GetResolvedReferencesStandalone(IEnumerable<s
             CSharp.Extractor.Analyse(stopwatch, analyser, options,
                 references => GetResolvedReferencesStandalone(referencePaths, references),
                 (analyser, syntaxTrees) => CSharp.Extractor.ReadSyntaxTrees(sources, analyser, null, null, syntaxTrees),
-                (syntaxTrees, references) => CSharpCompilation.Create("csharp.dll", syntaxTrees, references),
+                (syntaxTrees, references) => CSharpCompilation.Create(
+                    "csharp.dll", syntaxTrees, references, new CSharpCompilationOptions(OutputKind.ConsoleApplication, allowUnsafe: true)
```

Comment:
I was wondering whether specifying `OutputKind.ConsoleApplication` here might have any implications on the behaviour, but it seems that this is also the default value used by `CSharpCompilation.Create` if no `CSharpCompilationOptions` are given, so it should be all good.